bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -97,7 +97,7 @@ <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{
 
           <p>
           {% trans %}
-            Mitchell is a member of the Advisory Boards of the Oxford Internet Institute and the MIT Initiative on the Digital Economy. She is an MIT Media Lab Research Affiliate with the Open Agriculture Initiative, and a Board Member of OpenMRS, Inc., which develops open source medical records systems for use in resource-constrained environments. She co-chaired the U.S. Department of Commerce Digital Economy Board of Advisors from its inception in March 2016 until August 2017, served on the United Nations High Level Panel on Women’s Economic Empowerment, and the ICANN High Level Panel on Global Internet Cooperation and Governance Mechanisms. A native of the San Francisco Bay Area, Mitchell received her B.A. in Asian Studies from UC Berkeley and her J.D. from Berkeley Law.
+            Mitchell is a member of the Advisory Boards of the Oxford Internet Institute and the MIT Initiative on the Digital Economy. She is a Board Member of OpenMRS, Inc., which develops open source medical records systems for use in resource-constrained environments.
```

Comment:
I think we are deleting a little too much-

```suggestion
            Mitchell is a member of the Advisory Boards of the Oxford Internet Institute and the MIT Initiative on the Digital Economy. She is a Board Member of OpenMRS, Inc., which develops open source medical records systems for use in resource-constrained environments. She co-chaired the U.S. Department of Commerce Digital Economy Board of Advisors from its inception in March 2016 until August 2017, served on the United Nations High Level Panel on Women’s Economic Empowerment, and the ICANN High Level Panel on Global Internet Cooperation and Governance Mechanisms. A native of the San Francisco Bay Area, Mitchell received her B.A. in Asian Studies from UC Berkeley and her J.D. from Berkeley Law.
```

Reply:
You're right, I replaced the whole paragraph, not just the first sentence. I got sloppy 😞